lib/pact/command/command_builder.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Pact.Command.CommandBuilder do
```

Comment:
@EdwinGuayacan @Odraxs Please revert these changes. The word "Builder" is widely used in the OOP paradigm and elixir is a functional technology. And also you are missing the namespace "Kadena" in the module name.

My approach:
1. create a module Kadena.Pact.Command.Hash.
2. Create a `pull_unique/1` function.
3. Receive a list of signed commands as a parameter.

Reply:
@juanhurtado10 We leave the other functions in this new module too ? 
Because they are used for both commands ?

Reply:
@Odraxs No, this is a module with a single responsibility. Keep the other functions in their respective command files